src/main/resources/docs/multiple-tests/with-config/src/duplicate-keys.json

```diff
@@ -0,0 +1,7 @@
+//#Patterns: duplicate-keys
+{
+  "foo": "xpto",
+  //#Warning: duplicate-keys
```

Comment:
Can you remove the comments from the json files? Since comments are not part of json

Reply:
These files are now references to another files ✅